indra/newview/llluamanager.cpp

```diff
@@ -358,8 +351,8 @@ class LuaRemover
 // push the loaded module or throw a Lua error
 void LLRequireResolver::findModule()
 {
-    // If mPathToResolve is absolute, this replaces mSourceChunkname.parent_path.
-    auto absolutePath = (std::filesystem::path((mSourceChunkname)).parent_path() / mPathToResolve).u8string();
```

Comment:
I did a quick test locally (without this changeset), and, as Nicky-D said, unicode symbols in the path cause the issue. But changing this line to
` auto absolutePath = (std::filesystem::path((utf8str_to_utf16str(mSourceChunkname))).parent_path() / mPathToResolve).u8string();` fixes it.